pkgs/by-name/mf/mfcl5750dw/package.nix

```diff
@@ -0,0 +1,120 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  dpkg,
+  autoPatchelfHook,
+  makeWrapper,
+  perl,
+  gnused,
+  ghostscript,
+  file,
+  coreutils,
+  gnugrep,
+  which,
+}:
+
+let
+  arches = [
+    "x86_64"
+    "i686"
+    "armv7l"
+  ];
+
+  runtimeDeps = [
```

Comment:
This list should be added to `buildInputs`, and `buildInputs` should be used instead.